vertx-template-engines/vertx-web-templ-pebble/src/main/java/io/vertx/ext/web/templ/impl/PebbleVertxLoader.java

```diff
@@ -29,7 +29,7 @@
  *
  * @author Paulo Lopes
  */
-class PebbleVertxLoader implements Loader<String> {
+public class PebbleVertxLoader implements Loader<String> {
```

Comment:
Should this class moved out of the impl package? It used to be internal but now it can be used by the custom engine.